plugins/lookup/onepassword.py

```diff
@@ -495,28 +513,43 @@ def assert_logged_in(self):
         return False
 
     def full_signin(self):
-        required_params = [
-            "subdomain",
-            "username",
-            "secret_key",
-            "master_password",
-        ]
-        self._check_required_params(required_params)
+        if self.service_account_token:
+            environment_update = {"OP_SERVICE_ACCOUNT_TOKEN": self.service_account_token}
+            args = [
+                "whoami",
+            ]
 
-        args = [
-            "account", "add", "--raw",
-            "--address", "{0}.{1}".format(self.subdomain, self.domain),
-            "--email", to_bytes(self.username),
-            "--signin",
-        ]
+            return self._run(args, environment_update=environment_update)
+        else:
```

Comment:
These changes aren't needed since `full_signin` will be skipped if there is service token. If there is a problem with the service token, the lookup will exit before getting here.